src/routes/(proposals)/proposal/[slug]/+page.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  import PasswordPage from './password-page.svelte';
+  import ProposalPage from './proposal-page.svelte';
+  import ProposalAwards from './proposal-awards.svelte';
+  import ProposalClients from './proposal-clients.svelte';
+  import ProposalTestimonials from './proposal-testimonials.svelte';
+  import ProposalProjects from './proposal-projects.svelte';
+
+  export let data;
+
+  const proposal = data?.story?.content;
+  const awards = data?.services?.story?.content?.page[0].awards ?? [];
+  const projects = data?.home?.story.content.page[0].projects ?? [];
+  const testimonials = data?.services?.story?.content?.page[0].testimonials ?? [];
+  const clients = data?.services?.story?.content?.page[0].clients ?? [];
+</script>
+
```

Comment:
We are missing SEO, I think at least `title` should be added.
![image](https://github.com/significa/significa.co/assets/98809371/c772191a-20b2-4ae3-8557-2608ba5806be)